std_msgs/String request
---
baxter_core_msgs/EndpointState current_es
